<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>History</class>
 <widget class="QWidget" name="History">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>缴费记录</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 全局样式 */
QWidget {
    font-family: &quot;Inter&quot;, &quot;Microsoft YaHei&quot;, sans-serif;
    color: #334155;
    background-color: #f8fafc;
}

/* 标题区域 */
QLabel#label {
    font-size: 28px;
    font-weight: 700;
    color: #0f172a;
    padding: 20px 0 15px 24px;
    background-color: #ffffff;
    border-bottom: 1px solid #e2e8f0;
    text-shadow: 0 1px 2px rgba(0, 0, 0, 0.1);
}

/* 搜索区域卡片 */
QWidget#searchCard {
    background-color: #ffffff;
    border-radius: 12px;
    padding: 16px;
    margin: 16px;
    box-shadow: 0 4px 6px -1px rgba(0, 0, 0, 0.1), 0 2px 4px -1px rgba(0, 0, 0, 0.06);
}

/* 标签样式 */
QLabel {
    font-size: 14px;
    font-weight: 500;
    color: #475569;
    padding-right: 8px;
}

/* 输入框 */
QLineEdit {
    border: 1px solid #cbd5e1;
    border-radius: 6px;
    padding: 8px 12px;
    font-size: 14px;
    background-color: #ffffff;
    transition: all 0.2s ease;
}
QLineEdit:focus {
    border-color: #3b82f6;
    box-shadow: 0 0 0 3px rgba(59, 130, 246, 0.3);
    outline: none;
}

/* 查询按钮 */
QPushButton#queryButton {
    background: linear-gradient(135deg, #3b82f6, #2563eb);
    color: #ffffff;
    border: none;
    border-radius: 6px;
    padding: 9px 20px;
    font-size: 14px;
    font-weight: 600;
    transition: all 0.2s ease;
}
QPushButton#queryButton:hover {
    background: linear-gradient(135deg, #2563eb, #1d4ed8);
    transform: translateY(-1px);
    box-shadow: 0 4px 6px -1px rgba(59, 130, 246, 0.4);
}
QPushButton#queryButton:pressed {
    transform: translateY(0);
    box-shadow: 0 2px 4px -1px rgba(59, 130, 246, 0.4);
}

/* 表格卡片 */
QWidget#tableView {
    background-color: #ffffff;
    border-radius: 12px;
    margin: 16px;
    box-shadow: 0 4px 6px -1px rgba(0, 0, 0, 0.1), 0 2px 4px -1px rgba(0, 0, 0, 0.06);
}

/* 表格样式 */
QTableWidget {
    border: none;
    background-color: #ffffff;
    gridline-color: #e2e8f0;
    selection-background-color: #dbeafe;
    selection-color: #0f172a;
}
QTableWidget::item {
    padding: 10px 12px;
    border-bottom: 1px solid #e2e8f0;
}
QHeaderView::section {
    background-color: #f1f5f9;
    color: #334155;
    border: none;
    border-bottom: 2px solid #e2e8f0;
    padding: 10px 12px;
    font-weight: 600;
    text-align: left;
}

/* 状态列（假设是第5列） */
QTableWidget::item:nth-child(5) {
    color: #10b981; /* 薄荷绿 */
}

/* 生成发票按钮 */
QPushButton#generateInvoiceButton {
    background: linear-gradient(135deg, #10b981, #059669);
    color: #ffffff;
    border: none;
    border-radius: 6px;
    padding: 10px 24px;
    font-size: 15px;
    font-weight: 600;
    transition: all 0.2s ease;
}
QPushButton#generateInvoiceButton:hover {
    background: linear-gradient(135deg, #059669, #047857);
    transform: translateY(-1px);
    box-shadow: 0 4px 6px -1px rgba(16, 185, 129, 0.4);
}
QPushButton#generateInvoiceButton:pressed {
    transform: translateY(0);
    box-shadow: 0 2px 4px -1px rgba(16, 185, 129, 0.4);
}

/* 滚动条美化 */
QScrollBar:vertical {
    width: 8px;
    background: #f1f5f9;
    margin: 0;
    border-radius: 4px;
}
QScrollBar::handle:vertical {
    background: #94a3b8;
    border-radius: 4px;
}
QScrollBar::handle:vertical:hover {
    background: #64748b;
}
QScrollBar::add-line:vertical,
QScrollBar::sub-line:vertical {
    height: 0;
}

/* 底部区域 */
QWidget#bottomWidget {
    background-color: #f8fafc;
    border-top: 1px solid #e2e8f0;
    padding: 16px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Inter,Microsoft YaHei,sans-serif</family>
           <pointsize>-1</pointsize>
           <weight>87</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>缴费记录</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>Inter,Microsoft YaHei,sans-serif</family>
             <pointsize>-1</pointsize>
             <weight>62</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>缴费时间：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="paymentTimeEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>Inter,Microsoft YaHei,sans-serif</family>
             <pointsize>-1</pointsize>
             <weight>62</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>地址：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="addressEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="queryButton_2">
         <property name="font">
          <font>
           <family>Inter,Microsoft YaHei,sans-serif</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(6, 18, 71);
border-color: rgb(201, 246, 255);
background-color: rgb(216, 240, 255);</string>
         </property>
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Inter,Microsoft YaHei,sans-serif</family>
           <pointsize>-1</pointsize>
           <weight>62</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>备注：缴费方式：1为线上支付，0为线下支付</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tableView">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="generateInvoiceButton">
             <property name="font">
              <font>
               <family>Inter,Microsoft YaHei,sans-serif</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>生成电子发票</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
